src/query/settings/src/settings_getter_setter.rs

```diff
@@ -24,6 +24,12 @@ use crate::ReplaceIntoShuffleStrategy;
 use crate::ScopeLevel;
 use crate::SettingMode;
 
+#[derive(Clone, Copy)]
+pub enum FlightCompression {
+    Lz4,
+    Zstd,
+}
```

Comment:
```suggestion
pub enum FlightCompression {
    LZ4,
    ZSTD,
}
```

Reply:
This display will the enum?
<img width="903" alt="image" src="https://github.com/datafuselabs/databend/assets/172204/74619404-3a0d-412a-9ec9-64bd38fca860">

Reply:
From the settings show, user may set as:
```
set query_flight_compression='Lz4';
```
not 
```
set query_flight_compression='LZ4';
```